homeassistant/components/light/limitlessled.py

```diff
@@ -33,32 +37,54 @@
 from homeassistant.util.color import color_RGB_to_xy
 
 _LOGGER = logging.getLogger(__name__)
-REQUIREMENTS = ['ledcontroller==1.0.7']
+REQUIREMENTS = ['ledcontroller==1.1.0']
 
 
 def setup_platform(hass, config, add_devices_callback, discovery_info=None):
     """ Gets the LimitlessLED lights. """
     import ledcontroller
 
-    led = ledcontroller.LedController(config['host'])
+    # Handle old configuration format:
+    bridges = []
+    if 'bridges' in config:
+        bridges = config['bridges']
+    else:
+        bridges.append(config)
+
+    i = 0
+    for bridge in bridges:
+        bridge['id'] = i
+        i = i + 1
+
+    pool = ledcontroller.LedControllerPool([x['host'] for x in bridges])
 
     lights = []
-    for i in range(1, 5):
-        if 'group_%d_name' % (i) in config:
-            lights.append(LimitlessLED(led, i, config['group_%d_name' % (i)]))
+    for bridge in bridges:
```

Comment:
More Pythonic: instead of looping + setting the ID on the hash above, change this line to:
```python
for bridge_id, bridge in enumerate(bridges):
```